python/cudf/cudf/core/column/string.py

```diff
@@ -4297,6 +4297,20 @@ def to_pandas(self, index=None):
             pd_series.index = index
         return pd_series
 
+    @property
+    def values_host(self):
+        """
+        Return a numpy representation of the StringColumn.
+        """
+        return self.to_array()
```

Comment:
May want to make this `self.to_pandas().values` to be consistent as this will likely return a `pandas.arrays.StringArray` as of Pandas 1.0